examples/with-thirdparty-2fa-passwordless/src/SessionExpiredPopup/index.js

```diff
@@ -1,10 +1,10 @@
 import { useEffect } from "react";
-import { redirectToAuthWithoutRedirectToPath } from "supertokens-auth-react";
```

Comment:
If we are changing the example apps too during release, we should change all of their package.json as well to point to the github branch (otherwise releasing the repo will cause all the demo apps that are live to break)